geant3.sh

```
@@ -38,24 +38,10 @@ make ${JOBS:+-j $JOBS} install
 [[ ! -d $INSTALLROOT/lib64 ]] && ln -sf lib $INSTALLROOT/lib64
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)